File: src/main/java/com/tva/mk/dto/PayloadDto.java
package com.tva.mk.dto;

import com.fasterxml.jackson.annotation.JsonProperty;

public class PayloadDto {
	// region -- Fields --

	@JsonProperty(value = "id")
	private Integer id;

	@JsonProperty(value = "userName")
	private String userName;

	@JsonProperty(value = "firstName")
	private String firstName;

	@JsonProperty(value = "lastName")
	private String lastName;

	@JsonProperty(value = "accountNo")
	private String accountNo;

	@JsonProperty(value = "email")
	private String email;

	@JsonProperty(value = "contactNo")
	private String contactNo;

	@JsonProperty(value = "remarks")
	private String remarks;

	// end

	// region -- Get set --

	public Integer getId() {
		return id;
	}

	public void setId(Integer id) {
		this.id = id;
	}

	public String getUserName() {
		return userName;
	}

	public void setUserName(String userName) {
		this.userName = userName;
	}

	public String getFirstName() {
		return firstName;
	}

	public void setFirstName(String firstName) {
		this.firstName = firstName;
	}

	public String getLastName() {
		return lastName;
	}

	public void setLastName(String lastName) {
		this.lastName = lastName;
	}

	public String getEmail() {
		return email;
	}

	public void setEmail(String email) {
		this.email = email;
	}

	public String getContactNo() {
		return contactNo;
	}

	public void setContactNo(String contactNo) {
		this.contactNo = contactNo;
	}

	public String getRemarks() {
		return remarks;
	}

	public void setRemarks(String remarks) {
		this.remarks = remarks;
	}

	public String getAccountNo() {
		return accountNo;
	}

	public void setAccountNo(String accountNo) {
		this.accountNo = accountNo;
	}

	// end

	// region -- Methods --

	public PayloadDto() {

	}

	// end
}

File: dotnet/MoneyKeeper.Token/MoneyKeeper.Token/Dto/PayloadDto.cs
using Newtonsoft.Json;
using System;

namespace MoneyKeeper.Token.Dto
{
    /// <summary>
    /// Payload DTO
    /// </summary>
    public class PayloadDto
    {
        #region -- Properties --

        [JsonProperty("id")]

        public int Id { get; set; }

        [JsonProperty("userName")]

        public String UserName { get; set; }

        [JsonProperty("firstName")]

        public String FirstName { get; set; }

        [JsonProperty("lastName")]

        public String LastName { get; set; }

        [JsonProperty("accountNo")]

        public String AccountNo { get; set; }

        [JsonProperty("email")]

        public String Email { get; set; }

        [JsonProperty("contactNo")]

        public String ContactNo { get; set; }

        [JsonProperty("remarks")]

        public String Remarks { get; set; }

        [JsonProperty("uuid")]

        public Guid Uuid { get; set; }

        #endregion
    }
}